Tribler/Core/Session.py

```diff
@@ -412,18 +417,21 @@ def get_statistics(self):
     #
     # Persistence and shutdown
     #
-    def load_checkpoint(self, initialdlstatus=None, initialdlstatus_dict={}):
-        """ Restart Downloads from checkpoint, if any.
-
-        This method allows the API user to manage restoring downloads.
-        E.g. a video player that wants to start the torrent the user clicked
-        on first, and only then restart any sleeping torrents (e.g. seeding).
-        The optional initialdlstatus parameter can be set to DLSTATUS_STOPPED
-        to restore all the Downloads in DLSTATUS_STOPPED state.
-        The options initialdlstatus_dict parameter can be used to specify a
-        state overriding the initaldlstatus parameter per download id.
+    def load_checkpoint(self):
         """
-        self.lm.load_checkpoint(initialdlstatus, initialdlstatus_dict)
+        Restart Downloads from a saved checkpoint, if any. Note that we fetch information from the user download
+        choices since it might be that a user has stopped a download. In that case, the download should not be
+        resumed immediately when being loaded by libtorrent.
+        """
+        if not self.get_libtorrent():
```

Comment:
Shouldn't this be logged? Checkpointing without libtorrent is odd to begin with?